### PR TITLE
BNH-184 Update README

### PR DESCRIPTION
**Pull request by MaxTheComputerer**
- Updated readme with all the useful setup information from the Swiki space
- Copied over the database unit testing guide
- Merged in Ben's copyright notice branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,132 @@
...
-Or launch from your IDE, e.g. Rider.
+# Bricks and Hearts
+
+## Overview
+
+### Background
+
+ChangeAhead are an organisation that connects people in a vulnerable state with opportunities to get help and be
+empowered. One of the ways they do this is matching landlords with empty properties with people in need of housing
+assistance. The Bricks and Hearts project is a web platform to facilitate this. It currently consists of a landlord
```
Comment (emilypatsko) [inline README.md:9]: Bricks & Hearts 

I think we use an ampersand everywhere else :)

File: README.md [hunk 2]
```diff
@@ -1,25 +1,132 @@
...
+We also use the following APIs:
+
+* [Google OAuth 2.0](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-6.0)
+* [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/)
+* [Azure Maps API](https://docs.microsoft.com/en-us/azure/azure-maps/)
+* [Postcodes.io API](https://postcodes.io/)
+
+The staging and production sites are hosted on Microsoft Azure, and [GitHub Actions](https://docs.github.com/en/actions)
+is used for CI/CD.
+Credentials and other various ‘secrets’ are stored in [Keeper](https://keepersecurity.eu/vault/#) in the ‘Bricks&Hearts’
```
Comment (emilypatsko) [inline README.md:45]: They won't have access to our Keeper vault - we'll need to find another way to hand over these secrets.

Comment (emilypatsko) [reply README.md:45]: I would write the rest of the README under the assumption they have the relevant secrets _somewhere_

File: README.md [hunk 3]
```diff
@@ -1,25 +1,132 @@
...
+* C# on .NET Core 6
+  with the [ASP.NET Core MVC framework](https://docs.microsoft.com/en-us/aspnet/core/mvc/overview?view=aspnetcore-6.0)
+* MSSQL database and [Entity Framework Core 6](https://docs.microsoft.com/en-us/ef/core/)
+
+We also use the following APIs:
+
+* [Google OAuth 2.0](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?view=aspnetcore-6.0)
+* [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/)
+* [Azure Maps API](https://docs.microsoft.com/en-us/azure/azure-maps/)
+* [Postcodes.io API](https://postcodes.io/)
```
Comment (emilypatsko) [inline README.md:41]: Are there any credentials needed for these APIs which aren't in the Keeper vault?

**Review by emilypatsko — CHANGES_REQUESTED** [review]
Couple small things but otherwise looks really thorough

File: README.md (hunk 3 above)
Comment (MaxTheComputerer) [reply README.md:41]: I think they're all in the `appsettings.Development.json` file!

File: README.md (hunk 2 above)
Comment (MaxTheComputerer) [reply README.md]: Done :)

**Review by emilypatsko — APPROVED** [review]

**Pull request merged**